### PR TITLE
Configuration of static files path

### PR DESCRIPTION
**Pull request by thsutton**

### CODE REVIEW

File: examples/token-server.conf [hunk 1]
```diff
@@ -19,4 +19,5 @@ shibboleth {
 
 ui {
     page_size = 20
+    elephant = "./static/"
```
Comment (fractalcat) [inline examples/token-server.conf]: elephant?

File: lib/Network/OAuth2/Server/Configuration.hs [hunk 2]
```diff
@@ -39,15 +42,16 @@ import           Network.Wai.Middleware.Shibboleth as S
...
-        optVerifyRealm = "verify-token"
-        optShibboleth = S.defaultConfig
-    in ServerOptions{..}
+    let optDBString     = ""
+        optStatsHost    = "localhost"
+        optStatsPort    = 8888
+        optServiceHost  = "*"
+        optServicePort  = 8080
+        optUIPageSize   = (10 :: Integer) ^?! pageSize
+        optUIStaticPath = unsafePerformIO P.getDataDir </> "static"
```
Comment (fractalcat) [inline lib/Network/OAuth2/Server/Configuration.hs]: Is there a reason this needs to be unsafe?

Comment (thsutton) [inline lib/Network/OAuth2/Server/Configuration.hs]: This is supposed to be a complete, coherent, "working" default value. I'd probably prefer to remove it completely than make it a function or put it in `IO`.

Comment (fractalcat) [inline lib/Network/OAuth2/Server/Configuration.hs]: Fair enough, objection withdrawn.

**Pull request merged**